tests/testfiles/CommonMark_0.30/code_spans/006.test

```diff
@@ -1,12 +1,12 @@
 %   ---RESULT--- "example": 333,
 %   
-%   <p><code> b </code></p>
+%   <p><code> b </code></p>
 %   
 %   ---\RESULT---
 
 <<<
-` b `
+` b `
 >>>
 documentBegin
-codeSpan:  b 
+codeSpan:  b 
```

Comment:
Although the prose for [example 333][1] says:

> Only [spaces][2], and not [unicode whitespace][3] in general, are stripped in this way

the example contains ASCII spaces (U+0020), which should be stripped by a CommonMark-compliant processor. Therefore, I replaced the ASCII spaces in the example with non-breaking spaces (U+00A0). Perhaps we should report this to <https://github.com/commonmark/commonmark-spec>?

 [1]: https://spec.commonmark.org/0.30/#example-333
 [2]: https://spec.commonmark.org/0.30/#space
 [3]: https://spec.commonmark.org/0.30/#unicode-whitespace